canonical-web-and-design/ubuntu.com
# Updated Japanese UASD for storage and cleaned up old versions of docs

## Done

- Updated the japanese ua service description for the new storage limit
- Added the card to the japanese translation on the english uasd page
- Removed and renamed old legal docs from the pre markdown days

## QA

- download this branch
- go to http://0.0.0.0:8001/legal/ubuntu-advantage-service-description/
- see that it has a toc link to the japanese translation
- go to http://0.0.0.0:8001/legal/ubuntu-advantage-service-description/ja
- see that it has 48TB for ceph limits

## Note

- @solazio 	can you move the copy doc for japan into the https://drive.google.com/drive/folders/1sH5PYl1xSFSvkBz4f04bnf7SvvPtZAuz folder and then I can update it.
